funcbench/manifests/nodepools.yaml

```diff
@@ -0,0 +1,15 @@
+zone: {{ .ZONE }}
+projectid: {{ .PROJECT_ID }}
+cluster:
+  name: {{ .CLUSTER_NAME }}
+  nodepools:
+  # These node-pools will be deployed on triggering benchmark
+  - name: funcbench-{{ .PR_NUMBER }}
+    initialnodecount: 1
+    config:
+      machinetype: n1-highmem-4
```

Comment:
Is this the smalles machine that can be run as Sole-tenant nodes. If not please set to the lowest one possible to keep the cost of running these very low.